### PR TITLE
Add ability to get the git hash of build-commit

### PR DESCRIPTION
**Pull request by JoelAtDeluxe**
This PR embeds (but does not display) the git hash of the latest commit / build. Creating a PR to test the github workflow that does this. Presumably, this should be merged after there is a way to render this. As the menu renderings have a new PR up, I am avoiding conflicts until that is addressed first.

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by jkennedyvz — APPROVED** [review]
Hi Joel,

These changes look great.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -71,7 +71,7 @@ jobs:
       run: go test -v ./...
 
     - name: Build Executable
-      run: go build -o cmd/aterm/aterm cmd/aterm/*.go
+      run: go build -ldflags '-X main.gitHash=${{ needs.store-sha8.outputs.sha8 }}' -o cmd/aterm/aterm cmd/aterm/*.go
```
Comment (jrozner) [inline .github/workflows/ci.yaml:74]: This also needs to fill in the tag as well (assuming it's a tagged build).

**Review by jrozner — CHANGES_REQUESTED** [review]
I wrote a small go module to handle this https://github.com/jrozner/go-info and haven't gotten around to adding it. I don't care whether or not we use it but I would like the additional information made available (runtime version, build date). I'd also like to get the version information displayed whether it's on the command line or in the application itself (from a menu). If you do want to use that and there are issues with the library let me know. I'm sure there are also plenty of others available or we can just build the necessary code into the repo itself.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (JoelAtDeluxe) [reply .github/workflows/ci.yaml:74]: Yeah, this PR was more to handle just getting the info into the system. I can merge it into the PR we have for displaying updates, in addition to just rendering this info somewhere, either behind a flag, or as part of the startup. 

I'll look into  go-info to see if it's an easy fit into the project (I'm sure it is)

Comment (jrozner) [reply .github/workflows/ci.yaml:74]: I don't know that we need to merge this into the PR that adds updates but I would like to have a single PR that handles adding version + runtime information and displaying it. The update PR will need this information in place in order to accurately request updates.

**Review by jrozner — CHANGES_REQUESTED** [review]
Change looks good. It might make sense to make the go-info import use `.` so that the public variables are all imported to main just to make the build args a little less crazy remove needing to have the library specifically imported for the update notifications depending on where that code lives.

**Comment by JoelAtDeluxe** [comment]
I don't think doing a dot import on go-info will actually change the build args, or at least I couldn't get it to work.

**Review by jrozner — APPROVED** [review]

**Pull request merged**